Source/WTF/wtf/text/AtomStringImpl.h

```diff
@@ -82,8 +83,10 @@ class SUPPRESS_REFCOUNTED_WITHOUT_VIRTUAL_DESTRUCTOR AtomStringImpl final : publ
 
 inline RefPtr<AtomStringImpl> AtomStringImpl::lookUp(StringImpl* string)
 {
-    if (!string || string->isAtom())
-        return static_cast<AtomStringImpl*>(string);
+    if (!string)
+        return nullptr;
+    if (auto *atomImpl = dynamicDowncast<AtomStringImpl>(*string))
```

Comment:
nit: star on wrong side.

Reply:
Ugh... I keep mixing up the star position in WebKit & LLVM.

Reply:
I would have used the name `atom` without the impl suffix here and elsewhere.

Reply:
ok, changed to atom.